library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -149,8 +114,8 @@ public DownloadBatch retrieveBatchFor(FileDownloadInfo download) {
             int bigPictureUrlIndex = batchesCursor.getColumnIndexOrThrow(DownloadContract.Batches.COLUMN_BIG_PICTURE);
             int statusIndex = batchesCursor.getColumnIndexOrThrow(DownloadContract.Batches.COLUMN_STATUS);
             int visibilityColumn = batchesCursor.getColumnIndexOrThrow(DownloadContract.Batches.COLUMN_VISIBILITY);
-            int totalBatchSizeIndex = batchesCursor.getColumnIndexOrThrow(DownloadContract.Batches.COLUMN_TOTAL_BYTES);
-            int currentBatchSizeIndex = batchesCursor.getColumnIndexOrThrow(DownloadContract.Batches.COLUMN_CURRENT_BYTES);
+            int totalBatchSizeIndex = batchesCursor.getColumnIndexOrThrow(DownloadContract.BatchesWithSizes.COLUMN_TOTAL_BYTES);
```

Comment:
It's needed to maintain the same API, we can change it to use different URI to query batches with their sizes if it's worth it?